index.d.ts

```diff
@@ -0,0 +1,21 @@
+declare module "react-native-hyperlinked-text" {
+
+    import {TextStyle} from "react-native";
+
+    export interface ILinkDef {
+        regex: RegExp;
+        style?: TextStyle;
+        replaceText?: (orig: string, text: string, url: string) => string;
+        onPress?: (orig: string, text: string, url: string) => void;
```

Comment:
Also we have `noPress` property. An optional `boolean` that lets you override the global press handler with no handling.

Reply:
?